### PR TITLE
[Reviewed in priv repo] adding ipam, vnv and sec admin for 2024-01-01 preview

### PR DESCRIPTION
**Pull request by sssharma24**
…ed in priv repo

# ARM (Control Plane) API Specification Update Pull Request 

> [!TIP]
> Overwhelmed by all this guidance? See the `Getting help` section at the bottom of this PR description.

## PR review workflow diagram

Please understand this diagram before proceeding. It explains how to get your PR approved & merged.

spec_pr_review_workflow_diagram

## Purpose of this PR

What's the purpose of this PR? Check the specific option that applies. This is **mandatory**!

  - [ ] New resource provider.
  - [x] New API version for an existing resource provider. (If API spec is not defined in TypeSpec, the PR should have been created in adherence to [OpenAPI specs PR creation guidance](https://aka.ms/azsdkdocs/createopenapispec)).
  - [ ] Update existing version for a new feature. (This is applicable only when you are revising a private preview API version.)
  - [ ] Update existing version to fix OpenAPI spec quality issues in S360.
  - [ ] Convert existing [OpenAPI spec to TypeSpec spec](https://aka.ms/typespec/conversion) (do not combine this with implementing changes for a new API version).
  - [ ] Other, please clarify:
    - _edit this with your clarification_

## Due diligence checklist

To merge this PR, you **must** go through the following checklist and confirm you understood 
and followed the instructions by checking all the boxes:

- [x] I confirm this PR is modifying Azure Resource Manager (ARM) related specifications, and not data plane related specifications.
- [x] I have reviewed following [Resource Provider guidelines](https://aka.ms/rpguidelines), including
  [ARM resource provider contract](https://aka.ms/azurerpc) and
  [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md) (estimated time: 4 hours).  
  I understand this is required before I can proceed to the diagram Step 2, "ARM API changes review", for this PR.

## Additional information

<details>
<summary> Viewing API changes</summary>

For convenient view of the API changes made by this PR, refer to the URLs provided in the table 
in the `Generated ApiView` comment added to this PR. You can use ApiView to show API versions diff. 

</details>
<details>
<summary>Suppressing failures</summary>

If one or multiple validation error/warning suppression(s) is detected in your PR, please follow the 
[suppressions guide](https://aka.ms/azsdk/pr-suppressions) to get approval.

</details>

## Getting help

- First, please carefully read through this PR description, from top to bottom. Please fill out the `Purpose of this PR` and `Due diligence checklist`.
- If you don't have permissions to remove or add labels to the PR, request `write access` per [aka.ms/azsdk/access#request-access-to-rest-api-or-sdk-repositories](https://aka.ms/azsdk/access#request-access-to-rest-api-or-sdk-repositories)
- To understand what you must do next to merge this PR, see the `Next Steps to Merge` comment. It will appear within few minutes of submitting this PR and will continue to be up-to-date with current PR state.
- For guidance on fixing this PR CI check failures, see the hyperlinks provided in given failure 
  and https://aka.ms/ci-fix.
- For help with ARM review (PR workflow diagram Step 2), see https://aka.ms/azsdk/pr-arm-review.
- If the PR CI checks appear to be stuck in `queued` state, please add a comment with contents `/azp run`.
  This should result in a new comment denoting a `PR validation pipeline` has started and the checks should be updated after few minutes.
- If the help provided by the previous points is not enough, post to https://aka.ms/azsdk/support/specreview-channel and link to this PR.

### CODE REVIEW

**Comment by sssharma24** [comment]
New API changes have already been reviewed and approved by ARM team including the breaking changes and suppressions required during preview version: https://github.com/Azure/azure-rest-api-specs-pr/pull/17956. This PR adds the same swagger specs to stable version.

**Comment by ms-bogdan-rotaru** [comment]
@sssharma24 There are still failing checks, hence cannot merge the PR.

**Comment by sssharma24** [comment]
> @sssharma24 There are still failing checks, hence cannot merge the PR.

Swagger LintDiff issue for [BodyTopLevelProperties](https://github.com/Azure/azure-openapi-validator/blob/main/docs/body-top-level-properties.md) failures seem to be related to the issue https://github.com/Azure/azure-openapi-validator/issues/722. Can we apply suppression for these failures?

**Comment by ms-bogdan-rotaru** [comment]
@sssharma24 Cannot merge, 1 pending required reviewer is needed to approve the changes.

**Comment by sssharma24** [comment]
> @sssharma24 Cannot merge, 1 pending required reviewer is needed to approve the changes.

@markcowl could you please approve

**Review by ms-bogdan-rotaru — APPROVED** [review]

**Comment by ms-bogdan-rotaru** [comment]
@sssharma24 The branch is out-of-date and cannot be merged currently.

**Comment by rkmanda** [comment]
Pl get this approval and request a merge - https://github.com/Azure/azure-rest-api-specs/pull/29998#issuecomment-2253612157

File: specification/network/resource-manager/readme.md
Comment (raych1) [inline specification/network/resource-manager/readme.md:31]: This tag doesn't exist anymore. Can you update it to `package-2024-01-preview` either?

Comment (sssharma24) [reply specification/network/resource-manager/readme.md]: changing the tag is causing Avocado errors stating we have [MISSING_APIS_IN_DEFAULT_TAG](https://github.com/Azure/avocado/blob/master/README.md#MISSING_APIS_IN_DEFAULT_TAG)

Comment (sssharma24) [reply specification/network/resource-manager/readme.md]: package-2024-01 is the latest stable, so I don't think that's the issue reverting back to package-2024-01. example PR for when we merged an old preview version https://github.com/Azure/azure-rest-api-specs/commit/18ebda844331bc4b52babc60bd27112e4143f66c#diff-06b33c899bdfd2b8060eff3b065641a8f1b0ac7a8188d896d9cc2965ddfb22fe

Comment (raych1) [reply specification/network/resource-manager/readme.md]: The problem is `package-2024-01` tag definition doesn't exist in the readme.md that cause the input file cannot be found while generating the SDK.

**Pull request merged**
